src/panels/config/lovelace/dashboards/ha-config-lovelace-dashboards.ts

```diff
@@ -59,8 +60,41 @@ export class HaConfigLovelaceDashboards extends LitElement {
           sortable: true,
           filterable: true,
           direction: "asc",
+          template: (title, dashboard: any) => {
+            const titleTemplate = html`
+              ${title}
+              ${dashboard.default
+                ? html`
+                    <ha-icon
+                      style="padding-left: 10px;"
+                      icon="hass:check-circle-outline"
+                    ></ha-icon>
+                    <paper-tooltip>
+                      This is the default dashdoard.
+                    </paper-tooltip>
+                  `
+                : ""}
+            `;
+            return narrow
+              ? html`
+                  ${titleTemplate}
+                  <div class="secondary">
```

Comment:
I do wonder, why not have secondary in wide mode. I think that using double lines can result in nice looking tables like https://tailwindui.com/components/application-ui/tables/wide (but maybe for a future PR)

Reply:
But on a wide screen it looks weird if you have little data, but yeah, it can look good.